.gitattributes

```diff
@@ -2,5 +2,8 @@
 /.github/ export-ignore
 /.gitignore export-ignore
 /Doxyfile export-ignore
+/phive.xml
+/phpcs.xml export-ignore
```

Comment:
Do we have to have these files in the root dir? I would like to keep the root dir as tidy as possible.

Reply:
Concerning PHIVE: There is a second default location for the configuration file: `.phive/phars.xml`, and we could also move the tools from `tools/` to `.phive/` as well (like we do in the [Emogrifer project](https://github.com/MyIntervals/emogrifier)).

We can also move the PHP_Codesniffer configuration to file and that reference the configuration ("code standard") file in the Composer commands, e.g., to `config/phpcs.xml`. What do you think?

Reply:
I like it. Maybe phpunit.xml could also be moved to the config folder? And Doxyfile. Which reminds me: we should have a task that updates gh-pages after releases to publish https://sabberworm.github.io/PHP-CSS-Parser/.

Reply:
> We can also move the PHP_Codesniffer configuration to file and that reference the configuration ("code standard") file in the Composer commands, e.g., to `config/phpcs.xml`. What do you think?

A downside to that is that tools/editors will fail to discover the configuration files automatically.

Reply:
I'll move the PHIVE configuration file (and the PHAR symlinks) in this PR. We can take care of the other configuration files later then (in separate issues/PRs as needed).

Reply:
I have moved the PHARs and the PHIVE configuration file now and repushed.